ls8/cpu.c

```diff
@@ -42,29 +25,18 @@ void cpu_load(struct cpu *cpu, int argc, char** argv)
   }
   
   int address = 0;
-  // printf("here1\n");
   while (fgets(line, 1024, fp) != NULL) {
```

Comment:
you can use the magic of `endchar`
```
// Read all the lines and store them in RAM
  while (fgets(line, sizeof line, fp) != NULL)
  {

    // Convert string to a number
    char *endchar;
    unsigned char byte = strtol(line, &endchar, 2);
    ;

    // Ignore lines from which no numbers were read
    if (endchar == line)
    {
      continue;
    }

    // Store in ram
   ...
  }
```

Reply:
That fixes my empty line problems thanks!